### DISCUSSION TITLE
Remix node stacktraces without sourcemaps when using express and typescript

### DISCUSSION THREAD

**Discussion by madispuk** [remix-run/remix#4862] [Q&A]
I took the `templates/express` sample, but instead of having `server.js`, I changed it to `server.ts` because I need typescript. After that all the remix node error stack traces are without sourcemaps.
```
at splitIntoSubQueries (/Users/madis/dev/fyma/core-api-v2/build/index.js:1935:168)
at executeQuery (/Users/madis/dev/fyma/core-api-v2/build/index.js:1914:22)
```

When I get error in `server.ts` it has a correct stack trace with source map file names and line numbers, but when I get error in express action or loader, then it doesn't.

How can I make it so that remix includes source maps in it's node server errors?

**Comment by fatso83** [comment]
Ever got anywhere with this?
